.github/workflows/ci.yml

```diff
@@ -24,6 +24,21 @@ jobs:
 
       - name: Lint Ruby files and RBI shims
         run: bundle exec rubocop
+  rubocop_prism_experimental:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: "recursive"
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.3"
+          bundler-cache: true
+
+      - name: Lint Ruby files and RBI shims
+        run: bundle exec rubocop -c  .rubocop-prism-experimental.yml
```

Comment:
This is probably going to fail on every PR. Should we force this to exit with a zero status so that we avoid all PRs being red (and main)?

Reply:
Do you think there'd be that many? I was planning that we just add them as ignores in the YAML (like what I did for Layout/HeredocIndentation), then remove once resolved upstream.